front/src/common/types.ts

```diff
@@ -17,4 +17,4 @@ export const DATA_TYPES = {
 // type Minute = number;
 // type Second = number;
 
-export type TimeString = any; // `${Year}-${Month}-${Day}T${Hour}:${Minute}:${Second}`;
+export type TimeString = string | Date | undefined; // `${Year}-${Month}-${Day}T${Hour}:${Minute}:${Second}`;
```

Comment:
it’s only supposed to be a `string` not a `Date`

```suggestion
export type TimeString =  `${Year}-${Month}-${Day}T${Hour}:${Minute}:${Second}`;
```

Reply:
I do not understand, this does not parse